### PR TITLE
Update appveyor.yml

### PR DESCRIPTION
**Pull request by h404bi**
reference: [hexo/appveyor.yml](https://github.com/hexojs/hexo/blob/d1b762d665ee6df25e00b4863ebc26d917c1d573/appveyor.yml)

### CODE REVIEW

**Comment by JLHwung** [comment]
LGTM, let's wait for the appveyor results.

File: appveyor.yml [hunk 1]
```diff
@@ -1,22 +1,20 @@
 # Fix line endings in Windows. (runs before repo cloning)
 init:
   - git config --global core.autocrlf input
-  - git config --global user.email "you@example.com"
```
Comment (JLHwung) [inline appveyor.yml]: I never thought of that these `init` steps is test suite setup scripts.

Maybe we shall make more specific email/name to document that they are for test purpose.

It comes to me like
```bash
git config --global user.email "hexo-deployer-git-test@example.com"
- git config --global user.name "hexo-deployer-git Test"
```

Comment (h404bi) [reply appveyor.yml]: Ok, will use the specific email/name.

**Comment by h404bi** [comment]
All checks have passed now.

**Comment by JLHwung** [comment]
Awesome 🎉

**Pull request merged**